test.txt

```diff
@@ -0,0 +1 @@
+Testing
```

Comment:
![the-bean](https://user-images.githubusercontent.com/7034719/101063154-1cb5c200-3558-11eb-8d59-c8927234f6d7.jpg)